keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -218,6 +218,11 @@ def _forward(self, images, training=None):
         return box_pred, cls_pred
 
     def call(self, images, training=None):
+        if isinstance(images, tf.RaggedTensor):
+            raise ValueError(
+                "`RetinaNet` does not yet support `Ragged Tensor Input`"
```

Comment:
"`RetinaNet()` does not yet support inputs of type `RaggedTensor` for input images. "
"To correctly resize your images for object detection tasks, we recommend resizing using "
"`keras_cv.layers.Resizing(pad_to_aspect_ratio=True, bounding_box_format=your_format)` "
"on your inputs."

Reply:
Done !